### PR TITLE
feat: Add cancel sceduled calculation Wholesale API

### PR DESCRIPTION
**Pull request by ebbeknudsen**
## Description

- Add cancel sceduled calculation Wholesale API
- Call new Wholesale API in calculation mutation
  - This updates mutation input name from `guid` to `calculationId`
- Update Wholesale API client with new orchestration states
  - The frontend might need to be updated to use these states:
    - `Started` should be shown as calculating
    - `Canceled` should be shown as canceled

Requires merged Wholesale PR's:
- [x] https://github.com/Energinet-DataHub/opengeh-wholesale/pull/2697
- [x] https://github.com/Energinet-DataHub/opengeh-wholesale/pull/2686

## References

- https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-mosaic/210

### CODE REVIEW

File: apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs [hunk 1]
```diff
@@ -135,13 +139,15 @@ private static ProgressStatus GetScheduleProgressStatus(CalculationOrchestration
...
             CalculationOrchestrationState.Scheduled => ProgressStatus.Pending,
+            CalculationOrchestrationState.Started => ProgressStatus.Completed,
             CalculationOrchestrationState.Calculating => ProgressStatus.Completed,
             CalculationOrchestrationState.CalculationFailed => ProgressStatus.Completed,
             CalculationOrchestrationState.Calculated => ProgressStatus.Completed,
             CalculationOrchestrationState.ActorMessagesEnqueuing => ProgressStatus.Completed,
             CalculationOrchestrationState.ActorMessagesEnqueuingFailed => ProgressStatus.Completed,
             CalculationOrchestrationState.ActorMessagesEnqueued => ProgressStatus.Completed,
             CalculationOrchestrationState.Completed => ProgressStatus.Completed,
+            CalculationOrchestrationState.Canceled => ProgressStatus.Failed, // TODO: Is this correct?
```
Comment (ebbeknudsen) [inline apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs:150]: Is this correct @ManBearTM?

File: apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql [hunk 2]
```diff
@@ -708,7 +708,7 @@ input CalculationQueryInput {
 }
 
 input CancelScheduledCalculationInput {
-  guid: UUID!
+  calculationId: UUID!
```
Comment (ebbeknudsen) [inline apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql:711]: @ManBearTM there is a breaking change for the UI here - should we avoid it and instead keep the name as `guid`?

File: apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs [hunk 3]
```diff
@@ -68,13 +68,15 @@ protected override void Configure(IObjectTypeDescriptor<CalculationDto> descript
...
                 CalculationOrchestrationState.Scheduled => ProcessStatus.Neutral,
+                CalculationOrchestrationState.Started => ProcessStatus.Info,
                 CalculationOrchestrationState.Calculating => ProcessStatus.Info,
                 CalculationOrchestrationState.CalculationFailed => ProcessStatus.Danger,
                 CalculationOrchestrationState.Calculated => ProcessStatus.Info,
                 CalculationOrchestrationState.ActorMessagesEnqueuing => ProcessStatus.Info,
                 CalculationOrchestrationState.ActorMessagesEnqueuingFailed => ProcessStatus.Danger,
                 CalculationOrchestrationState.ActorMessagesEnqueued => ProcessStatus.Info,
                 CalculationOrchestrationState.Completed => ProcessStatus.Success,
+                CalculationOrchestrationState.Canceled => ProcessStatus.Warning,
```
Comment (ebbeknudsen) [inline apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs:79]: @ManBearTM I'm not sure if the `warning` color is correct to use here

File: apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql (hunk 2 above)
Comment (ManBearTM) [reply apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql:711]: No, seems like a good change 🙂

File: apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs (hunk 1 above)
Comment (ManBearTM) [reply apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs]: Changed it to neutral 👍

File: apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs (hunk 3 above)
Comment (ManBearTM) [reply apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Calculation/CalculationType.cs]: Don't know, but I went with neutral instead 😄

**Review by ManBearTM — APPROVED** [review]

**Pull request merged**